.github/workflows/ci.yml

```diff
@@ -75,8 +80,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        stack: ["2.5"]
-        ghc: ["8.10.4"]
+        stack: ["2.7"]
+        ghc: ["9.2.1"]
```

Comment:
I'm afraid, this won't work. Stack determines GHC version from the resolver snapshot in `stack.yaml`. You can bump up `8.10.4` to `8.10.7` here and use the [`lts-18.17`](https://www.stackage.org/lts-18.17) resolver in `stack.yaml`, which is currently the latest snapshot supporting GHC 8.10.7.